Loop/Managers/DeviceDataManager.swift

```diff
@@ -40,6 +40,8 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate, ReceiverDelegat
     /// Manages remote data (TODO: the lazy initialization isn't thread-safe)
     lazy var remoteDataManager = RemoteDataManager()
 
+    private var nightscoutDataManager: NightscoutDataManager!
```

Comment:
The high-level organization among the managers will change in the future. But for now, this keeps things separate.

`StatusTableViewController`, `WatchDataManager`, and `NightscoutDataManager` all are front-ends to Device and Loop data. I'd like to keep each one self-contained.